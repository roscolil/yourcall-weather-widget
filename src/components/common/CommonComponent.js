import React from 'react';

const CommonComponent = () => (
  <>
    <h1>Entando React Widget Boilerplate</h1>
  </>
);

export default CommonComponent;
