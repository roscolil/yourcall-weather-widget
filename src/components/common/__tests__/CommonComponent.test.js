import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import { render } from '@testing-library/react';
import CommonComponent from 'components/common/CommonComponent';

test('renders a message', () => {
  const { container, getByText } = render(<CommonComponent />);
  expect(getByText('Entando React Widget Boilerplate')).toBeInTheDocument();
  expect(container.firstChild).toMatchSnapshot();
});
