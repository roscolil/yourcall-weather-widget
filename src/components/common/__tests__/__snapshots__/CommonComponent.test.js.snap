// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<h1>
  Entando React Widget Boilerplate
</h1>
`;
